pyproject.toml

```diff
@@ -11,7 +11,7 @@ dynamic = ["version"]
 
 [project.optional-dependencies]
 
-gpunufft = ["gpuNUFFT>=0.9.0", "cupy-cuda12x"]
```

Comment:
Maybe we dont want this change. It got passed it from one of my commits, but which was later removed.